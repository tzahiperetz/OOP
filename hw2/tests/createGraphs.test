# Sample Testing file

# Create a graph
CreateGraph graph1
CreateGraph graph2
CreateGraph graph3
CreateGraph graph4
CreateGraph graph5
CreateGraph graph6
CreateGraph graph7
