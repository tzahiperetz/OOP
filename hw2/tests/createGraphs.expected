# Sample Testing file

# Create a graph
created graph graph1
created graph graph2
created graph graph3
created graph graph4
created graph graph5
created graph graph6
created graph graph7
